src/index.js

```diff
@@ -2,9 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import Login from './containers/login';
+import RegisterForm from './containers/registrationform';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<Login />, document.getElementById('root'));
+ReactDOM.render(<RegisterForm />, document.getElementById('root'));
```

Comment:
Don't do this. Start using the router and use `Login` or `RegisterForm` basis URL.